alidock/__init__.py

```diff
@@ -177,8 +177,14 @@ def entrypoint():
     argp = ArgumentParser()
     argp.add_argument("--quiet", dest="quiet", default=False, action="store_true",
                       help="Do not print any message")
-    argp.add_argument("--tmux", dest="tmux", default=False, action="store_true",
-                      help="Start or resume a detachable tmux session")
+
+    # tmux: both normal and terminal integration ("control mode")
+    tmuxArgs = argp.add_mutually_exclusive_group()
```

Comment:
@mconcas I have added this line to prevent both tmux arguments to be used at the same time.